client/scripts/stores/DraftStore.ts

```diff
@@ -0,0 +1,43 @@
+import IdStore from './IdStore';
+import { DiscussionDraft } from '../models';
+import { byAscendingCreationDate } from '../helpers';
+
+class DraftStore extends IdStore<DiscussionDraft> {
+  private _storeCommunity: { [identifier: string]: Array<DiscussionDraft> } = {};
```

Comment:
Should we call this draftsByCommunity or something like that for consistency with other stores?

Reply:
Yup, I'll update the tags store too while I'm at it, since that has the same pattern.